### DISCUSSION TITLE
绝区零有没有什么删除一天也不会被发现的东西

### DISCUSSION THREAD

**Discussion by momomomofish** [share121/inter-knot#4643] [灌水]
rt，想听听大家的看法

**Comment by sl44444** [comment]
嗷呜的一根毛

**Comment by lantainAirblue** [comment]
被以太侵蚀的月亮。自古绳匠不抬头

**Comment by ysdtal** [comment]
治安局朱鸢柜子上贴的喵吉便利贴

**Reply by HLXS-HLXS** [reply]
这个朱鸢应该会发现<img src="https://bbs-static.miyoushe.com/static/2024/03/21/36606bdafa3ea2537746b44f38bc5856_6388510732715759806.png" alt="丽都漫步-11号" width="75" >
